Islandora/islandora
# Collection view

**JIRA Ticket**: https://github.com/Islandora/documentation/issues/1336

# What does this Pull Request do?

Adds a collection view, a simple controller to let folks select what content type to create a new collection, and some tests around that controller (and two others).

# How should this be tested?

- Pull in this pr
- re-import `islandora_core_feature` to get the view
- go to `admin/content`
- you should see a tab for 'Collections', click on it
![image](https://user-images.githubusercontent.com/20773151/68966093-aeb72500-07b3-11ea-9310-34df77b6f3b7.png)
- you should see Collections on your site, their # of children, and disk usage
![image](https://user-images.githubusercontent.com/20773151/68966148-d27a6b00-07b3-11ea-9eba-08e51bb3620c.png)
- click 'Add Collection'
- You should be presented with a list of content types a la `/node/{nid}/members/add`
![image](https://user-images.githubusercontent.com/20773151/68966183-e4f4a480-07b3-11ea-9d99-710d3c0cd123.png)
- Click on 'Repository Item'
- You should get redirected to the creation form for a collection, with the model field prepopulated to have the 'Collection' term.
![image](https://user-images.githubusercontent.com/20773151/68966239-fb9afb80-07b3-11ea-9a82-6cf6c066f1c9.png)


# Additional Notes:
Is this sort of thing remotely useful?

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/8-x-committers
